_posts/0000-01-02-aram9321.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
“Life is not a problem to be solved, but a reality to be experienced.”– Soren Kierkegaard